src/api/aux.cpp

```diff
@@ -19,72 +29,65 @@ void parse_json_file(std::vector<Token> &tokens, std::vector<Order> &orders, std
     // TODO: Make input reading safe, i.e., handle all potential exceptions/errors
 
     // For-loop that reads all TOKEN information
-    
-    for (auto &i: json_file["tokens"].items()) {
-        boost::multiprecision::mpf_float t;
-        if (i.value()["external_price"].is_null())
-            t = -1;
-        else {
-            //std::string tt = std::to_string(i.value()["external_price"]);
-            std::string tt = (i.value()["external_price"]).dump();
-            t = static_cast<boost::multiprecision::mpf_float>(tt);
-        }
-        if (i.value()["alias"].is_null())
-            tokens.push_back(Token(i.key(), "_NULL", i.value()["decimals"], t));       // TODO: Check if multiple tokens with "null" alias can create any issue
+    for (auto const& tkn: json_file["tokens"].items()) {
```

Comment:
I think writing token instead of tkn is clearer and will it make easier to read for others.